src/middleware/isValidInvite.js

```diff
@@ -12,11 +12,11 @@ const verify = (req, res, next) => {
 
 	return jwt.verify(token, config('TOKEN_SECRET'), (err, user) => {
 
-		if (err) return next(new AppError('Invalid token', 403))
+		if (err) return next(new AppError('Invalid token', 401))
 
         
 		if (req.body.email !== user.email) 
-			return next(new AppError('Invalid request email', 403))
+			return next(new AppError('Invalid request email', 401))
```

Comment:
:smile:  i commented on just the earlier one because i felt the 403 you put here captured it in this case. 

credentials are valid but the email being different doesn't allow the request to continue or? @itzgeebee

Reply:
@jocrah Oh... Don't mind me. I'm like that sometimes

Reply:
haha no worries